modules/ROOT/pages/runtime-configure-eu.adoc

```diff
@@ -44,38 +45,39 @@ anypoint.platform.base_uri=https://eu1.anypoint.mulesoft.com
 anypoint.platform.analytics_base_uri=https://analytics-ingest.eu1.anypoint.mulesoft.com
 ----
 
-== Configuring Mule Runtime 3.x
+== Configuring Mule 3.x
 
 === wrapper.conf
 
-To configure `wrapper.conf` file, modify the following properties.
+To configure `wrapper.conf` file, modify the following properties:
```

Comment:
Consider making this sentence exactly the same as the one used in the same place for the Mule 4 text.
Use parallelism as much as possible.

Reply:
Got it - thank you!